node/libs/crypto/src/bls12_381/mod.rs

```diff
@@ -119,19 +132,20 @@ pub struct Signature(bls::Signature);
 
 impl Signature {
     /// Verifies a signature against the provided public key
-    pub fn verify(&self, msg: &[u8], pk: &PublicKey) -> Result<(), Error> {
+    pub fn verify(&self, msg: &[u8], pk: &PublicKey) -> anyhow::Result<()> {
         let result = self.0.verify(true, msg, DST, &[], &pk.0, true);
 
         match result {
             BLST_ERROR::BLST_SUCCESS => Ok(()),
-            err => Err(Error::SignatureVerification(err)),
+            err => Err(anyhow!("Signature verification failure: {err:?}")),
```

Comment:
`anyhow::format_err!`

Reply:
It seems to be just a reexport of `anyhow!` ([https://docs.rs/anyhow/latest/anyhow/index.html#reexports](https://docs.rs/anyhow/latest/anyhow/index.html#reexports)). Do you still think it makes sense to change it?